cognite_toolkit/_cdf_tk/data_classes/_module_directories.py

```diff
@@ -46,6 +47,13 @@ def parent_relative_paths(self) -> set[Path]:
         """All relative parent paths of the module."""
         return set(self.relative_path.parents)
 
+    @cached_property
+    def manifest(self) -> Manifest | None:
+        """The manifest of the module."""
+        if manifest_file := next(self.dir.glob("**/module.toml"), None):
```

Comment:
Should this be so flexible? Shouldn't we say that the manifest have to be on the first level of the module folder and not deep inside some subfolder?

Reply:
Agree, it shouldn't. Tightening up.